rmgpy/molecule/inchi.py

```diff
@@ -602,13 +602,13 @@ def create_augmented_layers(mol):
     else:
         molcopy = mol.copy(deep=True)
 
+        rdkitmol = to_rdkit_mol(molcopy, remove_h=True)
+        _, auxinfo = Chem.MolToInchiAndAuxInfo(rdkitmol, options='-SNon')  # suppress stereo warnings
+
```

Comment:
Is Codecov correct that lines #L605 - L606 were not covered by tests ? Does that mean this whole method is not tested? (or is codecov not right?)

Reply:
This is something weird. The failures in the unit tests testing this function inspired my change here. I looked at the [codecov coverage profile](https://app.codecov.io/gh/ReactionMechanismGenerator/RMG-Py/pull/2545/blob/rmgpy/molecule/inchi.py) and found it reported 0% coverage of the whole inchi.py, but RMG actually has [unit tests](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/main/test/rmgpy/molecule/inchiTest.py) for it. @JacksonBurns any thoughts?

Reply:
So this test is definitely running (you can search for it in the CI logs) and the new code (and this whole file) are being run, so CodeCov is just wrong. From brief googling I can't find a solution, so I will debug this separately, but this PR should be fine.